spec/cell_spec.rb

```diff
@@ -0,0 +1,38 @@
+require './lib/cell'
+require './lib/grid'
+
+describe Cell do
+
+  let(:cell) {Cell.new('1')}
+  let(:cell0) {Cell.new('0')}
+  let(:grid) {Grid.new('005000009009000000000000000000000000000000000000000000000000000000000000000000000')}
+
+  it "should contain a value" do
+    expect(cell.value).to eq(1)
+  end
+
+  it "should raise error if does not contain a value between 0-9" do
+    expect(lambda{Cell.new('10')}).to raise_error('Values must be less than 10!')
```

Comment:
Line is too long. [81/80]<br>Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.